sfcasts/metadata.yml

```diff
@@ -1 +1,3 @@
-chapters: []
+chapters:
+    - intro
+    - command-intro
```

Comment:
@MolloKhan Btw, the actual file name is `sfcasts/command_intro.md`, i.e. with the underscore instead of dash... I think we should match style in both cases, probably `sfcasts/command-intro.md` is the correct one.

Reply:
yep, I already fixed that. Thanks anyways